### PR TITLE
Adds critical as a state to the crew monitor

### PR DESCRIPTION
**Pull request by warior4356**
## What Does This PR Do
This PR makes it so crew is shown as in Critical when they are below the health threshold to enter soft crit (0 hp).

## Why It's Good For The Game
Currently, there is no distinction between hard crit and unconscious, and it would be beneficial to point to people who are in danger of dying with a clear identifier. 

## Images of changes
image

## Changelog
:cl:
tweak: Added critical as a state to the crew monitor
/:cl:

### CODE REVIEW

**Review by AffectedArc07 — APPROVED** [review]
Waiting on Headmin approval

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**
